contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -504,6 +513,7 @@ contract StakeManager is IStakeManager, StakeManagerStorage {
         delete signerToValidator[validators[validatorId].signer];
         signerToValidator[_signer] = validatorId;
         validators[validatorId].signer = _signer;
+        // latestSignerUpdateEpoch[validatorId] = latestSignerUpdateEpoch[validatorId].add(signerUpdateLimit);
```

Comment:
this is required you need to set it to current checkpoint count.

Reply:
thanks